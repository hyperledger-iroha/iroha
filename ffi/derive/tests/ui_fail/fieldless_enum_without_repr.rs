use iroha_ffi::FfiType;

/// Enum
#[derive(FfiType)]
pub enum FfiStruct {
    Variant1,
    Variant2
}

fn main() {}
