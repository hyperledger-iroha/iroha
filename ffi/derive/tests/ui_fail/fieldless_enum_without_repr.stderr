error: Fieldless enum missing #[repr(int)]
 --> tests/ui_fail/fieldless_enum_without_repr.rs:5:10
  |
5 | pub enum FfiStruct {
  |          ^^^^^^^^^
